src/onnxflow/justbuildit/ignition.py

```diff
@@ -290,6 +277,9 @@ def load_or_make_state(
     model: build.UnionValidModelInstanceTypes = None,
     inputs: Optional[Dict[str, Any]] = None,
     quantization_samples: Optional[Collection] = None,
+    state_type: Type = build.State,
+    cache_validation_func: Callable = validate_cached_model,
+    extra_state_args: Optional[Dict] = None,
```

Comment:
Gives flexibility to pass in additional args to `State`. For example, `groqflow` will pass in `use_sdk`.